### PR TITLE
RAINCATCH-1072 remove logger documentation

### PR DESCRIPTION
**Pull request by austincunningham**
**Motivation:**
Logger module has changed update the documentation to reflect this.

**What**
update logger documentation and remove unused parts

- [x] update documentation

### CODE REVIEW

File: docs/workforce-management-framework/topics/pro-enabling-logging.adoc [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+Supported implementations:
+
+|====
+| Implementation | Usage
+
+|BunyanLogger |Implements Bunyan and is used for server side logging
+|ClientLogger |Implements LogLevel and is used for client (web and mobile) logging
+|====
+
```
Comment (jstaffor) [inline docs/workforce-management-framework/topics/pro-enabling-logging.adoc]: Should we included sample of Console too (It's ok to say "no" - you know best (ie: John, no, a dev really does not need a console sample))

[source,javascript]
----
// Import logger implementation
import {ConsoleLogger, logger, setLogger} from '@raincatcher/logger';

// Instaniate logger
const log = new ConsoleLogger();;
setLogger(log);
----
+
. To log a comment, add the following code _after_ the code directly above:
+
[source,javascript]
----
logger.info('This log will render with ConsoleLogger');
----

Comment (austincunningham) [reply docs/workforce-management-framework/topics/pro-enabling-logging.adoc]: console is gone replaced with clientlogger

Comment (jstaffor) [reply docs/workforce-management-framework/topics/pro-enabling-logging.adoc]: Should we included sample of clientlogger too?

Comment (austincunningham) [reply docs/workforce-management-framework/topics/pro-enabling-logging.adoc]: there is a link to the example code which contains clientlogger example

@ https://github.com/feedhenry-raincatcher/raincatcher-core/blob/master/cloud/logger/example/index.ts

Comment (wtrocki) [reply docs/workforce-management-framework/topics/pro-enabling-logging.adoc]: No.. We should take different angle here and in all documentations. 
We should provide mostly why this modules is used, what value it gives to user and how user should interact with it. This pattern applies to all module docs. 
We should minimize source code as you see we already changed a lot of names etc.

Things getting outdated so fast that we may change them 5-6 times before releasing docs.

Comment (jstaffor) [reply docs/workforce-management-framework/topics/pro-enabling-logging.adoc]: "We should provide mostly why this modules is used, what value it gives to user"
== <[con-enabling-logging.adoc](https://raw.githubusercontent.com/redhat-documentation/modular-docs/master/files/TEMPLATE_CONCEPT_concept_template_and_guidelines.adoc)>

"and how user should interact with it."
== <[pro-enabling-logging.adoc](https://raw.githubusercontent.com/redhat-documentation/modular-docs/master/files/TEMPLATE_PROCEDURE_doing_one_procedure.adoc)>

"This pattern applies to all module docs." +10000000

"We should minimize source code as you see we already changed a lot of names etc." +1

**Comment by wtrocki** [comment]
This PR got outdated and no longer reflects our vision. Going to close it. 
We will reuse some of the contents from this PR.

**Comment by wtrocki** [comment]
@austincunningham - This change can be considered as cleanup change to remove contents that should never land on the master.

**Review by wtrocki — APPROVED** [review]

**Pull request merged**